### PR TITLE
Use --help for stylish haskell

### PR DESCRIPTION
**Pull request by tolysz**
Also take only the first line of version output

### CODE REVIEW

**Comment by tolysz** [comment]
OSX timeout builds

File: src/com/haskforce/settings/HaskellToolsConfigurable.java [hunk 1]
```diff
@@ -194,7 +194,7 @@ public void updateVersion() {
             if (pathText.isEmpty()) {
                 versionField.setText("");
             } else {
-                versionField.setText(getVersion(pathText, versionParam));
+                versionField.setText(getVersion(pathText, versionParam).split("\r\n|\r|\n")[0]);
```
Comment (carymrobbins) [inline src/com/haskforce/settings/HaskellToolsConfigurable.java]: Note that this is unsafe for two reasons: `getVersion` might return `null` and `split` might return an empty array.

I've appended a commit to resolve these issues - 0c3fca3

**Comment by carymrobbins** [comment]
I went ahead and rebased your changes and merged into master.  Thanks!

**Comment by tolysz** [comment]
Thank you!

**Pull request closed without merging**